docs/quickstart.md

```diff
@@ -0,0 +1,108 @@
+# Goose in 5 minutes
+
+## Quickstart guide
+
+Goose is a developer agent that supercharges your software development by automating an array of coding tasks directly within your terminal or IDE. You can get it setup and running on your devices in only a few minutes.
+
+### Installation
+
+To install Goose, use `pipx`. First ensure [pipx][pipx] is installed:
+
+``` sh
+brew install pipx
+pipx ensurepath
+```
+
+Then install Goose:
+
+```sh
+pipx install goose-ai
+```
+
+### Running Goose
+
+#### Start a session
+
+From your terminal, navigate to the directory you'd like to start from and run:
+
+```sh
+goose session start
+```
+
+#### Set up a provider
+Goose works on top of LLMs and by default, it uses `openai` as the LLM provider but you can customize it as needed. You'll be prompted to set an [OPENAI_API_KEY][openai-key] if you haven't set one previously.
+
+>[!TIP]
+> **Billing:**
+>
+> You will need to add credits to your Open AI accounts to be able to successfully make requests.
+>
+
+
+#### Make Goose do the work for you
+You will see the Goose prompt `G❯`:
+
+```
+G❯ type your instructions here exactly as you would tell a developer.
```

Comment:
```suggestion
G❯ type your instructions here exactly as you would speak to a developer.
```

Reply:
I wonder if we can give them an example prompt to run so they can get up and running right away here.